doc/manual/plugins.rst

```diff
@@ -46,7 +50,6 @@ Global plugins
    plugins_global/errors
    plugins_global/rc
    plugins_global/wcsmatch
-   plugins_global/changehistory
```

Comment:
> modify our sphinx build to extract and create the RST subpages

I am not sure if this is possible without too much Sphinx-fu. I'll have to investigate.